packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -400,6 +406,25 @@ export const MSTeamsNotifier: Notifier = {
 
   async integrationUpdated(dataLoader: DataLoaderWorker, teamId: string, userId: string) {
     ;(await getMSTeams(dataLoader, teamId, userId))?.integrationUpdated()
+  },
+
+  async standupResponseSubmitted(
+    dataLoader: DataLoaderWorker,
+    meetingId: string,
+    teamId: string,
+    userId: string
+  ) {
+    const {meeting, team} = await loadMeetingTeam(dataLoader, meetingId, teamId)
+    const user = await dataLoader.get('users').load(userId)
+    const responses = await getTeamPromptResponsesByMeetingId(meetingId)
```

Comment:
+1 can we parallelize these 3 calls + maybe that `getMSTeams`?

Reply:
Ooh yea nice catch! Fixed.

I left the `getMSTeams` call (and similar calls for other integrations) as-is since the prevailing pattern seems to be "get data -> check data -> get integration"